src/GoogleMaps.xml

```diff
@@ -130,5 +138,27 @@
             <category>Map controls</category>
             <description>Enable/disable Zoom control</description>
         </property>
+        <property key="defaultMakerIcon" type="image" required="false">
+            <caption>Default marker icon</caption>
+            <category>Advanced</category>
+            <description>The default image to use if no enumeration is available.</description>
+        </property>
+        <property key="markerImages" type="object" isList="true" required="false">
```

Comment:
1) Update `readme.md` with this new feature

2) Please note that the bade say Mendix 7.6, while dependencies  claims 7.4 (not sure if that is right, but at least it is not consistent)

3) with the e2e test added, the developers comments are not up to date, please use the general development.md file similar to 
https://github.com/FlockOfBirds/checkbox-filter/blob/feature/MVP/development.md
(do not forget to change the URLs)